extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/ClusterRoleBindingConfig.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkus.kubernetes.deployment;
+
+import java.util.Map;
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class ClusterRoleBindingConfig {
+
+    /**
+     * Name of the ClusterRoleBinding resource to be generated. If not provided, it will use the application name plus the role
+     * ref name.
+     */
+    @ConfigItem
+    public Optional<String> name;
+
+    /**
+     * Labels to add into the RoleBinding resource.
+     */
+    @ConfigItem
+    public Map<String, String> labels;
+
+    /**
+     * The name of the Role resource to use by the RoleRef element in the generated ClusterRoleBinding resource.
+     */
+    @ConfigItem
+    public String roleName;
+
+    /**
+     * If the Role sets in the `role-name` property is cluster wide or not.
+     */
+    @ConfigItem(defaultValue = "true")
+    public Boolean clusterWide;
```

Comment:
I left it because theoretically, we can bind the ClusterRoleBinding with a namespace-based Role resource.
But thinking again about it, I don't think it makes sense. So, will remove it.